### PR TITLE
Add w2v2bert to pipeline

### PR DESCRIPTION
**Pull request by ylacombe**
# What does this PR do?
https://github.com/huggingface/transformers/pull/28165 introduced a new W2V2-based model that uses a different feature extractor than classic CTC-based models.
In particular, it takes mel-spectrograms as `input_features`, instead of raw waveform as `input_values`.

The pipeline only takes `input_values` for this kind of models, which requires a bit of workaround.

Note that I've also run every slow tests from the ASR pipeline, just to make sure.

cc @amyeroberts @sanchit-gandhi 

Also cc @ydshieh, it would be nice to add the model architecture to the tiny model repo !

### CODE REVIEW

**Review by amyeroberts — APPROVED** [review]
Great - all LGTM. Thanks for adding! 

Let's wait for a second approval from @ydshieh to confirm the pipeline tests are good. 

Once that's done and all slow ASR tests pass, happy to merge!

**Comment by ydshieh** [comment]
> Also cc @ydshieh, it would be nice to add the model architecture to the tiny model repo !

OK, will update next week (if it succeed to create the tiny model though)

File: tests/pipelines/test_pipelines_automatic_speech_recognition.py
Comment (ydshieh) [inline tests/pipelines/test_pipelines_automatic_speech_recognition.py:303]: I would love to have a short description of what this test is doing (than the above `test_torch_large`).

By reading this block:
- I don't know what `fbank` is and/or if this test is testing `fbank` or other stuff
  - from the PR description, `fbank` is not mentioned, that's why I get a bit lost

File: tests/pipelines/test_pipelines_automatic_speech_recognition.py
Comment (ydshieh) [inline tests/pipelines/test_pipelines_automatic_speech_recognition.py:309]: > it takes mel-spectrograms as input_features, instead of raw waveform as input_values

Is `wavefrom` the best name here? It looks a bit contradicted to the PR description.

Not a big deal, but if a more proper name is available, we can use it then.

**Comment by ydshieh** [comment]
> second approval

nothing big from my side other than 2 tiny comments.

File: tests/pipelines/test_pipelines_automatic_speech_recognition.py
Comment (ylacombe) [reply tests/pipelines/test_pipelines_automatic_speech_recognition.py:303]: I'll cat it `test_torch_large_with_input_features`

File: tests/pipelines/test_pipelines_automatic_speech_recognition.py
Comment (ylacombe) [reply tests/pipelines/test_pipelines_automatic_speech_recognition.py:309]: The input is a waveform but is processed into a melspectrogram (fbank) ! I'll keep this name

Comment (xenova) [reply tests/pipelines/test_pipelines_automatic_speech_recognition.py:309]: On that note, I see docstring still says:
> input_features (`torch.FloatTensor` of shape `(batch_size, sequence_length)`):
            Float values of input raw speech waveform.

https://github.com/huggingface/transformers/blob/708b19eb093bcfb4efcfa229925b249397c2bd5e/src/transformers/models/wav2vec2_bert/modeling_wav2vec2_bert.py#L1020-L1025

**Pull request merged**
